lib/CodeTransform/Tests/Adapter/WorseReflection/Refactor/fixtures/generateMutator1.test

```diff
@@ -0,0 +1,20 @@
+// File: source
+
+<?php
+
+class generateMethod
+{
+    pr<>ivate $method;
+}
+// File: expected
+<?php
+
+class generateMethod
+{
+    private $method;
+
+    public function method($method): void
```

Comment:
I think it would be very rare to have a non-prefixed mutator? so not sure this should be the default

Reply:
The default prefix is `set` see https://github.com/phpactor/phpactor/pull/1946/files#diff-0a820ddea77da22f468ce7dc570c241bd40a8b2f918314da9a1e9c36fdfe0c1cR112

I can change the tests so they all use `set` as a prefix though

Reply:
Ah OK, no worries it's fine